Sdk/Speckle.Connectors.Utils/Operations/ProxyKeys.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Speckle.Connectors.Utils.Operations;
+
+public static class ProxyKeys
```

Comment:
This seems higher effort than just typing a class for the root collection (imo)

Reply:
This is TBD, do not know where we are with formalizing root object @didimitrie